x-pack/test/functional/services/aiops/explain_log_rate_spikes_analysis_table.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ProvidedType } from '@kbn/test';
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+type ExpectedSectionTableEntries = Record<string, string>;
+export interface ExpectedSectionTable {
+  section: string;
+  expectedEntries: ExpectedSectionTableEntries;
+}
+
+export type AnalyticsTableRowDetails = Record<'jobDetails', ExpectedSectionTable[]>;
+
+export type ExplainLogRateSpikesAnalysisTable = ProvidedType<
+  typeof ExplainLogRateSpikesAnalysisTableProvider
+>;
+
+export function ExplainLogRateSpikesAnalysisTableProvider({
+  getPageObject,
```

Comment:
We don't use `getPageObject` at the moment in this service, so we could remove it from here.

Reply:
Removed in [1f9f454](https://github.com/elastic/kibana/pull/138661/commits/1f9f45402238398da7a4830013a4bb6ba09a5950).